examples/main.go

```diff
@@ -35,7 +44,18 @@ func main() {
 			return
 		}
 		// Log raw message struct
-		fmt.Println(mes)
+		//fmt.Println(mes)
```

Comment:
I meant forever (as a replacement for reading from bot.Incoming until there is nil), but I think that is probably best done outside the bot, where we now (in the example) would read from bot.Incoming to keep the bot running.